### ISSUE TITLE
Allow for responsive scaling of game canvas

### ISSUE THREAD

**Issue by W-A-James** [ceramic-mug/snak3#5]

**Comment by ceramic-mug** [comment]
Yes. In each timestep check for rescaled window? CSS rescaling and reinitiation of game with same states?

**Comment by ceramic-mug** [comment]
CSS is now automatically rescaling canvas (50% width and height, which is default setting when canvas is created), but the game within the canvas is rendered at the scale and resolution of the original window/canvas. What do we want to do about this? Changing the canvas size really does change the nature of the playing field since the game itself is all about space, so we need a good fix for this. Maybe a CSS resize is good enough, since the game isn't long, or a dialogue popup when someone rescales the canvas that they need to refresh the page

**Comment by W-A-James** [comment]
@ceramic-mug I checked and when you resize the window when the game starts, it doesn't rescale the game window since that auto-scaling only happens once. I say we figure out a way to keep a fixed aspect ratio and maybe even scale the other elements accordingly.

**Comment by W-A-James** [comment]
I think what I'm going to do is keep a 4:3 ratio, setting the width according to the window width and the height gets its value from the aspect ratio.

**Comment by W-A-James** [comment]
@ceramic-mug See new [branch](https://github.com/ceramic-mug/snak3/tree/allow-for-responsive-scaling-of-game-canvas) and [pull request](https://github.com/ceramic-mug/snak3/pull/9)
